ouroboros-consensus/docs/report/chapters/future/genesis.tex

```diff
@@ -1499,3 +1500,178 @@ \subsection{Possible optimisations}
 to not being up to date would require careful thought.
 
 \end{itemize}
+
+\section{Suboptimal chain selection}
+
+The Ouroboros papers assume, by way of simplification, that nodes broadcast
+entire chains to each other. Every node collects all of these chains, and
+every slot selects the best chain they saw. In addition, the Ouroboros analysis
+assumes a constant $\Delta$, such that chains are submitted to all nodes in
+the network within $\Delta$ slots. A consequence of $\Delta > 1$ is that
```

Comment:
Why clarify that `\Delta > 1`? I guess `=0` makes no sense, but still.

Reply:
Delta could be something between 0 and 1. But since we know it must be bigger then one slot, and slot is 1 second, we can assume > 1 second

Reply:
If `Delta = 1`, every block makes it in one slot, so it always arrives at the next slot leader in time.